WcaOnRails/app/webpacker/components/CompetitionResults.js

```diff
@@ -85,15 +54,21 @@ function EventResults({ competitionId, eventId }) {
   return (
     <div className="event-results">
       {data.rounds.map((round) => (
-        <RoundResultsTable key={round.id} round={round} eventName={data.name} eventId={data.id} />
+        <RoundResultsTable
+          key={round.id}
+          round={round}
+          competitionId={competitionId}
+          adminMode={adminMode}
+        />
       ))}
     </div>
   );
 }
 
-function CompetitionResults({ competitionId }) {
+function CompetitionResults({ competitionId, canAdminResults }) {
   const { loading, error, data } = useLoadedData(competitionApiUrl(competitionId));
   const [selectedEvent, setSelectedEvent] = useState(null);
+  const [adminMode, setAdminMode] = useState(false);
```

Comment:
It was the original behavior but Jacob raised a fair point that it may not be the desirable view when viewing results "casually" as someone who can admin them (which actually makes sense).
So I'd say we should keep that to false.

Reply:
Okay, fair point. Thanks for the clarification!